### PR TITLE
[FLINK-4159] Remove Quickstart exclusions for unused dependencies

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -207,15 +206,9 @@ under the License.
 									<exclude>com.esotericsoftware.kryo:kryo</exclude>
 									<exclude>com.esotericsoftware.minlog:minlog</exclude>
 									<exclude>org.objenesis:objenesis</exclude>
-									<exclude>com.twitter:chill_*</exclude>
```
Comment (greghogan) [inline flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml]: This is a dependency of `flink-runtime` which is a dependency of `flink-dist`.

File: flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -207,15 +206,9 @@ under the License.
...
 									<exclude>com.twitter:chill-java</exclude>
-									<exclude>com.twitter:chill-avro_*</exclude>
-									<exclude>com.twitter:chill-bijection_*</exclude>
-									<exclude>com.twitter:bijection-core_*</exclude>
-									<exclude>com.twitter:bijection-avro_*</exclude>
 									<exclude>commons-lang:commons-lang</exclude>
 									<exclude>junit:junit</exclude>
-									<exclude>de.javakaffee:kryo-serializers</exclude>
```
Comment (greghogan) [inline flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml]: This is a dependency of `flink-core` which is a dependency of `flink-dist`.

File: flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml (hunk 1 above)
Comment (greghogan) [inline flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml]: I also see that this was not in the list in the Jira for FLINK-4159.

Comment (zentol) [inline flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml]: you're right, must've slipped in.

Comment (zentol) [inline flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml]: I've reverted the removal.

**Comment by greghogan** [comment]
+1 to merge

**Comment by zentol** [comment]
merging

**Pull request closed without merging**